docs/install_guides/pypi-openvino-rt.md

```diff
@@ -78,6 +78,27 @@ If installation was successful, you will not see any error messages (no console
 
 For general troubleshooting steps and issues, see [Troubleshooting Guide for OpenVINO Installation](https://docs.openvino.ai/2022.2/openvino_docs_get_started_guide_troubleshooting.html). The following sections also provide explanations to several error messages. 
 
+### Errors with Installing via PIP for Users in China
+
+Users in China might encounter errors while downloading sources via PIP during OpenVINO™ installation. To resolve the issues, try one of the following options:
+   
+* Add the download source using the ``-i`` parameter with the Python ``pip`` command. For example: 
+
+   ``` sh
+   pip install openvino-dev -i https://mirrors.aliyun.com/pypi/simple/
```

Comment:
```suggestion
   pip install openvino -i https://mirrors.aliyun.com/pypi/simple/
```

Reply:
Actually I'm not quite sure if it works for runtime. Should we have someone test it?

Reply:
openvino-dev includes openvino runtime as a dependency so if it works for openvino-dev, it should work for runtime too. It would be good if someone can test this from China.

Reply:
I tried from my local, and it worked. I think it's OK.